datastore/datastore_record.go

```diff
@@ -35,17 +35,20 @@ type StoredMetricRecord struct {
 
 	// CounterStartTime is used to keep start timestamp for cumulative metrics.
 	CounterStartTime time.Time
+
+	// Storage manager
```

Comment:
This comment doesn't provide any information. Remove it or fix it.

Reply:
Agreed. Removed.